src/Engine/ProtoCore/Lang/CallSite.cs

```diff
@@ -1121,18 +1121,32 @@ private FunctionGroup FirstFunctionGroupInInheritanceChain(RuntimeCore runtimeCo
             int typeID = svThisPtr.metaData.type;
 
             //Test for exact match
-            List<FunctionEndPoint> exactFeps = new List<FunctionEndPoint>();
+            List<FunctionEndPoint> exactFeps = null;
```

Comment:
if we're just going to set this directly, why use a list and waste time calling ToList()?